### PR TITLE
fix(core/title): handle line break in title

### LINKED ISSUE #3798: The title of the generated document is inconsistent with the original title
## Important info

- URL to affected spec: https://w3c.github.io/clreq/
- ReSpec version: 26.13.4
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

**What happened (e.g., it crashed)?:**

The `title` of the static HTML document generated by ReSpec is not the same as the original `title` of the document.

The [title](https://github.com/w3c/clreq/blob/3384e9a8747baa2b5b0703a0041465c528f1cdd8/index.html#L5) of https://w3c.github.io/clreq/ is "Requirements for Chinese Text Layout - 中文排版需求", but in the generated HTML it is "Requirements for Chinese Text Layout中文排版需求", and Pubrules reports an error:

> Content of title and h1 do not match. Text of title is 'Requirements for Chinese Text Layout中文排版需求' while h1 is (transformed into) 'Requirements for Chinese Text Layout - 中文排版需求'.

**Expected behavior (e.g., it shouldn't crash):**

ReSpec should probably not modify the `title` of the document.

**Optional, steps to reproduce:**

1. Open https://w3c.github.io/clreq/
2. Generate a static HTML document
3. Open the document and view the `title` of the document

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #3798 

Title becomes: 
Screen Shot 2021-10-05 at 3 53 39 pm

### CODE REVIEW

File: src/core/title.js [hunk 1]
```diff
@@ -55,13 +55,18 @@ export function run(conf) {
 }
 
 function setDocumentTitle(conf, h1Elem) {
+  debugger;
```
Comment (marcoscaceres) [inline src/core/title.js:58]:
```suggestion
```

File: src/core/title.js [hunk 2]
```diff
@@ -60,8 +60,12 @@ function setDocumentTitle(conf, h1Elem) {
   if (!h1Elem.isConnected) {
     h1Elem.textContent = document.title || `${l10n.default_title}`;
   }
-
-  let documentTitle = norm(h1Elem.textContent);
+  // We replace `<br>` with ":" and "-", as appropriate.
```
Comment (marcoscaceres) [inline src/core/title.js:63]:
```suggestion
  // We replace ":<br>" with ":", and "<br>" with "-", as appropriate.
```

**Review by xfq — APPROVED** [review]
LGTM. Thank you!

**Comment by xfq** [comment]
Note that this is not just for localized doc titles, some other specs (likes [webauthn-2](https://www.w3.org/TR/2021/REC-webauthn-2-20210408/)) also have titles that contain breaking lines.

**Comment by marcoscaceres** [comment]
It might be a few days until we get this out. Still making Process 2021 changes.

**Pull request merged**